inline_entity_form.module

```diff
@@ -58,30 +62,18 @@ function inline_entity_form_get_controller(FieldDefinitionInterface $instance) {
 
 /**
  * Implements hook_entity_info_alter().
+ *
+ * @TODO - Add handlers for other core entity types.
  */
-function inline_entity_form_entity_info_alter(&$entity_info) {
-  /**
-   * @var \Drupal\Core\Entity\EntityType[] $entity_info
-   */
-  $entity_info['node']->setFormClass('inline entity form', '\Drupal\inline_entity_form\Plugin\InlineEntityForm\NodeInlineEntityFormController');
+function inline_entity_form_entity_type_build(array &$entity_types) {
+  /** @var \Drupal\Core\Entity\EntityTypeInterface[] $entity_types */
+  $entity_types['node']->setFormClass('inline entity form', '\Drupal\inline_entity_form\InlineEntityForm\NodeInlineEntityFormHandler');
```

Comment:
I think Node module is optional. We should wrap this and all other core entity types in isset().